src/components/DocRoot/index.js

```diff
@@ -160,6 +160,9 @@ export default function DocRoot() {
           <ul>
             <li><NavLink className="doc-section" to="/docs/guide/httpSenderResilience4jRetry">HttpSender with Resilience4j retry</NavLink></li>
           </ul>
+          <ul>
+            <li><NavLink className="doc-section" to="/docs/guide/customMeterRegistry">Custom metery registry</NavLink></li>
```

Comment:
```suggestion
            <li><NavLink className="doc-section" to="/docs/guide/customMeterRegistry">Custom meter registry</NavLink></li>
```

Reply:
@shakuzen Thanks! I applied the fix in 7fcfc25.